packages/clay-modal/src/Context.ts

```diff
@@ -10,6 +10,12 @@ import {Status} from './types';
 export interface IContext {
 	ariaLabelledby?: string;
 
+	/**
+	 * Flag indicating if the modal should be the clean variant
+	 * with the title hidden
+	 */
+	clean?: boolean;
```

Comment:
I couldn't find an example of a modal with an image in the Figma document. The way it looks is similar to Cards. We can add a status type called image at https://github.com/liferay/clay/blob/master/packages/clay-modal/src/types.ts#L6 instead of `clean` here. It will add the class `modal-image` to `modal-dialog`.

Reply:
Looks good, I think it's a good idea to do it from the status. Thanks!